docs/data/toolpad/connecting-to-datasources/postgreSQL.md

```diff
@@ -0,0 +1,31 @@
+# PostgreSQL
+
+<p class="description">PostgreSQL datasource allows us to fetch data straight from PostgreSQL database.</p>
+
+## Working with PostgreSQL
+
+As explained in the [connections](/toolpad/connecting-to-datasources/connections/) section before you can use **PostgreSQL** query you will first need to setup **connection**:
+
+1. Once your **connection** is ready click on **ADD QUERY** in the **Inspector** on the right.
+
+1. Select **PostgreSQL** datasource and click **CREATE QUERY**:
+
+   ![Postgres type](/static/toolpad/postgres-query-1.png)
+
+1. We got all the basic configuration settings as described in [queries](/toolpad/connecting-to-datasources/queries/) section.
+
+1. In addition we can configure following properties inline:
+
+   ![Postgres configuration](/static/toolpad/postgres-query-2.png)
+
+   - On the left side we have a textarea where we can pass **SQL** query.
+
+   - On the right side we can query results.
+
+   - Below the query editor we can configure **Parameters** which can be used to bind data for our SQL queries. We can use the standard PostgreSQL `$1`, `$2`, `$3` interpolation.
+
+   ![Parameters configuration](/static/toolpad/postgres-query-3.png)
+
+   Like so: `SELECT * FROM "db" WHERE name = $name`.
```

Comment:
Would use a codeblock for this. Do we have SQL syntax highlighting?

```suggestion
   Example: 
   ```sql
   SELECT * FROM "db" 
   WHERE name = $name`;
   ```
```

Reply:
I tried it first but I got error that sql is not supported

Reply:
I see, looks like syntax highlighting is handled [here](https://github.com/mui/material-ui/blob/512896973499adbbda057e7f3685d1b23cc02de9/docs/packages/markdown/prism.js). Perhaps we can continue this PR without the `sql` language tag. And then look into supporting it as a separate effort in the core docs?

Reply:
Added that to my todo list of follow ups for docs